scripts/add_grouper_groups

```diff
@@ -3,19 +3,16 @@
 from os import path
```

Comment:
I was wondering why you have these two imports from `os` separated in different lines.

Reply:
Fixed